qiskit_experiments/calibration/calibrations.py

```diff
@@ -0,0 +1,438 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Class to store the results of a calibration experiments."""
+
+import copy
+import dataclasses
+from collections import namedtuple
+from datetime import datetime
+from typing import Any, Dict, Set, Tuple, Union, List, Optional
+
+from qiskit.providers.backend import BackendV1 as Backend
+from qiskit.circuit import Gate
+from qiskit import QuantumCircuit
+from qiskit.pulse import Schedule, DriveChannel, ControlChannel, MeasureChannel
+from qiskit.pulse.channels import PulseChannel
+from qiskit.circuit import Parameter
+from qiskit_experiments.calibration.exceptions import CalibrationError
+from qiskit_experiments.calibration.parameter_value import ParameterValue
+
+ParameterKey = namedtuple("ParameterKey", ["schedule", "parameter"])
+
+
+class Calibrations:
+    """
+    A class to manage schedules with calibrated parameter values.
+    Schedules are stored in a dict and are intended to be fully parameterized,
+    including the index of the channels. The parameter values are stored in a
+    dict where parameters are keys.
+    """
+
+    def __init__(self, backend: Backend):
```

Comment:
This seems to indicate a more general lack of functionality when it comes to creating backend objects.

Reply:
I changed the logic so that `Calibrations` does not need a backend. It is intended to work with schedules etc. I introduced a subclass `BackendCalibrations` that is intended to work more tightly with `backend` objects. See here: 0a01a7a.